src/state/get-droppable-over.js

```diff
@@ -26,26 +26,35 @@ const getRequiredGrowth = memoizeOne((
   // some items in it, but not enough to completely fill its size.
   // In this case - when the droppable already contains excess space - we
   // don't need to add the full placeholder size.
+
+  const getResult = (existingSpace: number): ?Position => {
+    // this is the space required for a placeholder
+    const requiredSpace: number = draggable.page.withMargin[droppable.axis.size];
+
+    if (requiredSpace <= existingSpace) {
+      return null;
+    }
+    const requiredGrowth: Position = patch(droppable.axis.line, requiredSpace - existingSpace);
+
+    return requiredGrowth;
+  };
+
   const dimensions: DraggableDimension[] = getDraggablesInsideDroppable(droppable, draggables);
 
+  // Droppable is empty
   if (!dimensions.length) {
-    return null;
+    const existingSpace: number = droppable.page.withMargin[droppable.axis.size];
```

Comment:
While writing tests I came across this: we should add space to empty `Droppables` if required. I also added some tests for this